StardewArchipelago/GameModifications/Modded/JunimoShopStockModifier.cs

```diff
@@ -71,17 +72,17 @@ private void GenerateBlueJunimoStock(ShopData shopData)
             var fishData = DataLoader.Fish(Game1.content);
             shopData.Items.Clear();
 
-            foreach (var fish in fishData)
+            foreach (var (id, fishInfo) in fishData)
             {
                 string[] fishSeasons = null;
-                if (fishData[fish.Key].Split("/")[1] != "trap")
+                if (fishData[id].Split("/")[1] != "trap")
```

Comment:
the whole point of deconstruction is that you can use fishInfo here, instead of fishData[id]